src/databricks_docs_synthetic.yaml

```diff
@@ -346,22 +359,23 @@
             \ could previously access."
         source: []
         gradings:
-        -   judge_name: human
+        -   judge_name: liang.zhang@
             criteria:
             -   name: retrieval
                 rating: 0
                 reason: we don't know what is retrieved
                 tags: []
             -   name: answer
                 rating: 1
-                reason: helpful
+                reason: This is a perfect answer extraction the relevant sentences
+                    from the ground truth chunk!
```

Comment:
This means we can use GPT-3.5 to extract truly relevant sentences from a ground-truth chunk!